### ISSUE TITLE
Sublime Build 3103 HTML Highlighting

### ISSUE THREAD

**Issue by Solanar** [squ1b3r/Djaneiro#101]
Sublime Text 3 Build 3103 seems to have broken highlighting for HTML (Django).
![djaneiro-highlight-bud](https://cloud.githubusercontent.com/assets/819536/13062738/e617dfbe-d3fd-11e5-8647-4a3b6d1fe3c2.PNG)

**Comment by squ1b3r** [comment]
Just checked and everything seems to be okay. No other issues from other users. So I guess there is something off with your setup.

**Comment by Solanar** [comment]
I disabled all packages except Djaneiro and tried again with built in colour scheme All Hallow's Eve:
![djaneiro-highlight-bug2](https://cloud.githubusercontent.com/assets/819536/13068085/d9deb5ce-d434-11e5-94d8-db499aa9f73e.PNG)
Same issue, something in this script block is breaking something. The combination of `<script></div></script>` seems to cause it. I think it's because it expects javascript and a `/` is part of `</script>` or a js regex.  
This is a [knockout template](http://knockoutjs.com/documentation/template-binding.html).  
I love using Djaneiro, it's been a massive help to my development. I'm not sure what changed as it was working before the new Sublime build, but I hope you'll take another look.

**Comment by squ1b3r** [comment]
Yeah, I see the problem now. I need to rewrite syntax file at some point. 3103 introduced a new syntax file mechanic which can help with this. Have absolutely no idea when I might be able to get to it tho.

**Comment by Solanar** [comment]
I've sort of got around it by including a django comment with a slash in it: `{# / #}`, but it's not ideal.  
I see in the code (and tried it out) that `type="text/template"` has the correct highlighting. Would adding `type="text/html"` there be an easier fix until you have more time? I think the regex would become `(?!.*type=["']text/(template|html)['"])`.

**Comment by gunsmoke** [comment]
+1, having the same issue, but only when i use <scripts> as a template holder.

**Comment by ptim** [comment]
I found that a 'missing' trailing semi in a script tag can throw it off, too:
```html
<script>WebFontConfig={custom:{families:['MarrSans','PublicoText']},timeout: 2000}</script>`
